improver/lapse_rate.py

```diff
@@ -44,6 +43,8 @@
 from improver.utilities.cube_checker import spatial_coords_match
 from improver.utilities.cube_manipulation import (
     enforce_coordinate_ordering, get_dim_coord_names)
+from improver.utilities import neighbourhood_tools
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No name 'neighbourhood_tools' in module 'improver.utilities'](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5319684)